CRITICAL - 2024-09-05 15:43:06 --> syntax error, unexpected token "public", expecting end of file
#0 C:\xampp\htdocs\reportes_cursos\system\Autoloader\Autoloader.php(241): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp\\htdocs...')
#1 C:\xampp\htdocs\reportes_cursos\system\Autoloader\Autoloader.php(219): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers...')
#2 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers...')
#3 C:\xampp\htdocs\reportes_cursos\system\CodeIgniter.php(797): class_exists('\\App\\Controller...', true)
#4 C:\xampp\htdocs\reportes_cursos\system\CodeIgniter.php(406): CodeIgniter\CodeIgniter->startController()
#5 C:\xampp\htdocs\reportes_cursos\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\reportes_cursos\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2024-09-05 15:43:13 --> syntax error, unexpected token "public", expecting end of file
#0 C:\xampp\htdocs\reportes_cursos\system\Autoloader\Autoloader.php(241): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp\\htdocs...')
#1 C:\xampp\htdocs\reportes_cursos\system\Autoloader\Autoloader.php(219): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers...')
#2 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers...')
#3 C:\xampp\htdocs\reportes_cursos\system\CodeIgniter.php(797): class_exists('\\App\\Controller...', true)
#4 C:\xampp\htdocs\reportes_cursos\system\CodeIgniter.php(406): CodeIgniter\CodeIgniter->startController()
#5 C:\xampp\htdocs\reportes_cursos\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\reportes_cursos\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2024-09-05 15:45:43 --> Undefined variable $hash
#0 C:\xampp\htdocs\reportes_cursos\app\Controllers\Usuarios.php(153): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined varia...', 'C:\\xampp\\htdocs...', 153)
#1 C:\xampp\htdocs\reportes_cursos\system\CodeIgniter.php(832): App\Controllers\Usuarios->insertar()
#2 C:\xampp\htdocs\reportes_cursos\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
#3 C:\xampp\htdocs\reportes_cursos\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\reportes_cursos\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2024-09-05 15:58:25 --> syntax error, unexpected token "private"
#0 C:\xampp\htdocs\reportes_cursos\system\Autoloader\Autoloader.php(241): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp\\htdocs...')
#1 C:\xampp\htdocs\reportes_cursos\system\Autoloader\Autoloader.php(219): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers...')
#2 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers...')
#3 C:\xampp\htdocs\reportes_cursos\system\CodeIgniter.php(797): class_exists('\\App\\Controller...', true)
#4 C:\xampp\htdocs\reportes_cursos\system\CodeIgniter.php(406): CodeIgniter\CodeIgniter->startController()
#5 C:\xampp\htdocs\reportes_cursos\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\reportes_cursos\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
ERROR - 2024-09-05 16:08:57 --> ErrorException: pg_query(): Query failed: ERROR:  no existe la relación «usuarios»
LINE 2: FROM &quot;usuarios&quot;
             ^ in C:\xampp\htdocs\reportes_cursos\system\Database\Postgre\Connection.php:135
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'C:\\xampp\\htdocs...', 135)
#1 C:\xampp\htdocs\reportes_cursos\system\Database\Postgre\Connection.php(135): pg_query(Resource id #135, 'SELECT 1\nFROM "...')
#2 C:\xampp\htdocs\reportes_cursos\system\Database\BaseConnection.php(670): CodeIgniter\Database\Postgre\Connection->execute('SELECT 1\nFROM "...')
#3 C:\xampp\htdocs\reportes_cursos\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1\nFROM "...')
#4 C:\xampp\htdocs\reportes_cursos\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM "...', Array, false)
#5 C:\xampp\htdocs\reportes_cursos\system\Validation\Rules.php(142): CodeIgniter\Database\BaseBuilder->get()
#6 C:\xampp\htdocs\reportes_cursos\system\Validation\Validation.php(292): CodeIgniter\Validation\Rules->is_unique('TOMS8485', 'usuario', Array, NULL)
#7 C:\xampp\htdocs\reportes_cursos\system\Validation\Validation.php(157): CodeIgniter\Validation\Validation->processRules('usuario', 'usuario', 'TOMS8485', Array, Array)
#8 C:\xampp\htdocs\reportes_cursos\system\Controller.php(160): CodeIgniter\Validation\Validation->run()
#9 C:\xampp\htdocs\reportes_cursos\app\Controllers\Usuarios.php(142): CodeIgniter\Controller->validate(Array)
#10 C:\xampp\htdocs\reportes_cursos\system\CodeIgniter.php(832): App\Controllers\Usuarios->insertar()
#11 C:\xampp\htdocs\reportes_cursos\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
#12 C:\xampp\htdocs\reportes_cursos\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#13 C:\xampp\htdocs\reportes_cursos\public\index.php(37): CodeIgniter\CodeIgniter->run()
#14 {main}
CRITICAL - 2024-09-05 16:08:57 --> Call to a member function getRow() on bool
#0 C:\xampp\htdocs\reportes_cursos\system\Validation\Validation.php(292): CodeIgniter\Validation\Rules->is_unique('TOMS8485', 'usuario', Array, NULL)
#1 C:\xampp\htdocs\reportes_cursos\system\Validation\Validation.php(157): CodeIgniter\Validation\Validation->processRules('usuario', 'usuario', 'TOMS8485', Array, Array)
#2 C:\xampp\htdocs\reportes_cursos\system\Controller.php(160): CodeIgniter\Validation\Validation->run()
#3 C:\xampp\htdocs\reportes_cursos\app\Controllers\Usuarios.php(142): CodeIgniter\Controller->validate(Array)
#4 C:\xampp\htdocs\reportes_cursos\system\CodeIgniter.php(832): App\Controllers\Usuarios->insertar()
#5 C:\xampp\htdocs\reportes_cursos\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
#6 C:\xampp\htdocs\reportes_cursos\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\reportes_cursos\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
ERROR - 2024-09-05 16:22:49 --> ErrorException: pg_query(): Query failed: ERROR:  no existe la relación «usuarios»
LINE 2: FROM &quot;usuarios&quot;
             ^ in C:\xampp\htdocs\reportes_cursos\system\Database\Postgre\Connection.php:135
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'C:\\xampp\\htdocs...', 135)
#1 C:\xampp\htdocs\reportes_cursos\system\Database\Postgre\Connection.php(135): pg_query(Resource id #135, 'SELECT 1\nFROM "...')
#2 C:\xampp\htdocs\reportes_cursos\system\Database\BaseConnection.php(670): CodeIgniter\Database\Postgre\Connection->execute('SELECT 1\nFROM "...')
#3 C:\xampp\htdocs\reportes_cursos\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1\nFROM "...')
#4 C:\xampp\htdocs\reportes_cursos\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM "...', Array, false)
#5 C:\xampp\htdocs\reportes_cursos\system\Validation\Rules.php(142): CodeIgniter\Database\BaseBuilder->get()
#6 C:\xampp\htdocs\reportes_cursos\system\Validation\Validation.php(292): CodeIgniter\Validation\Rules->is_unique('TOMG847584', 'usuario', Array, NULL)
#7 C:\xampp\htdocs\reportes_cursos\system\Validation\Validation.php(157): CodeIgniter\Validation\Validation->processRules('usuario', 'usuario', 'TOMG847584', Array, Array)
#8 C:\xampp\htdocs\reportes_cursos\system\Controller.php(160): CodeIgniter\Validation\Validation->run()
#9 C:\xampp\htdocs\reportes_cursos\app\Controllers\Usuarios.php(142): CodeIgniter\Controller->validate(Array)
#10 C:\xampp\htdocs\reportes_cursos\system\CodeIgniter.php(832): App\Controllers\Usuarios->insertar()
#11 C:\xampp\htdocs\reportes_cursos\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
#12 C:\xampp\htdocs\reportes_cursos\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#13 C:\xampp\htdocs\reportes_cursos\public\index.php(37): CodeIgniter\CodeIgniter->run()
#14 {main}
CRITICAL - 2024-09-05 16:22:49 --> Call to a member function getRow() on bool
#0 C:\xampp\htdocs\reportes_cursos\system\Validation\Validation.php(292): CodeIgniter\Validation\Rules->is_unique('TOMG847584', 'usuario', Array, NULL)
#1 C:\xampp\htdocs\reportes_cursos\system\Validation\Validation.php(157): CodeIgniter\Validation\Validation->processRules('usuario', 'usuario', 'TOMG847584', Array, Array)
#2 C:\xampp\htdocs\reportes_cursos\system\Controller.php(160): CodeIgniter\Validation\Validation->run()
#3 C:\xampp\htdocs\reportes_cursos\app\Controllers\Usuarios.php(142): CodeIgniter\Controller->validate(Array)
#4 C:\xampp\htdocs\reportes_cursos\system\CodeIgniter.php(832): App\Controllers\Usuarios->insertar()
#5 C:\xampp\htdocs\reportes_cursos\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
#6 C:\xampp\htdocs\reportes_cursos\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\reportes_cursos\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
ERROR - 2024-09-05 16:26:06 --> ErrorException: pg_query(): Query failed: ERROR:  no existe la relación «usuarios»
LINE 2: FROM &quot;usuarios&quot;
             ^ in C:\xampp\htdocs\reportes_cursos\system\Database\Postgre\Connection.php:135
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'C:\\xampp\\htdocs...', 135)
#1 C:\xampp\htdocs\reportes_cursos\system\Database\Postgre\Connection.php(135): pg_query(Resource id #135, 'SELECT 1\nFROM "...')
#2 C:\xampp\htdocs\reportes_cursos\system\Database\BaseConnection.php(670): CodeIgniter\Database\Postgre\Connection->execute('SELECT 1\nFROM "...')
#3 C:\xampp\htdocs\reportes_cursos\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1\nFROM "...')
#4 C:\xampp\htdocs\reportes_cursos\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM "...', Array, false)
#5 C:\xampp\htdocs\reportes_cursos\system\Validation\Rules.php(142): CodeIgniter\Database\BaseBuilder->get()
#6 C:\xampp\htdocs\reportes_cursos\system\Validation\Validation.php(292): CodeIgniter\Validation\Rules->is_unique('tomf', 'usuario', Array, NULL)
#7 C:\xampp\htdocs\reportes_cursos\system\Validation\Validation.php(157): CodeIgniter\Validation\Validation->processRules('usuario', 'usuario', 'tomf', Array, Array)
#8 C:\xampp\htdocs\reportes_cursos\system\Controller.php(160): CodeIgniter\Validation\Validation->run()
#9 C:\xampp\htdocs\reportes_cursos\app\Controllers\Usuarios.php(142): CodeIgniter\Controller->validate(Array)
#10 C:\xampp\htdocs\reportes_cursos\system\CodeIgniter.php(832): App\Controllers\Usuarios->insertar()
#11 C:\xampp\htdocs\reportes_cursos\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
#12 C:\xampp\htdocs\reportes_cursos\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#13 C:\xampp\htdocs\reportes_cursos\public\index.php(37): CodeIgniter\CodeIgniter->run()
#14 {main}
CRITICAL - 2024-09-05 16:26:06 --> Call to a member function getRow() on bool
#0 C:\xampp\htdocs\reportes_cursos\system\Validation\Validation.php(292): CodeIgniter\Validation\Rules->is_unique('tomf', 'usuario', Array, NULL)
#1 C:\xampp\htdocs\reportes_cursos\system\Validation\Validation.php(157): CodeIgniter\Validation\Validation->processRules('usuario', 'usuario', 'tomf', Array, Array)
#2 C:\xampp\htdocs\reportes_cursos\system\Controller.php(160): CodeIgniter\Validation\Validation->run()
#3 C:\xampp\htdocs\reportes_cursos\app\Controllers\Usuarios.php(142): CodeIgniter\Controller->validate(Array)
#4 C:\xampp\htdocs\reportes_cursos\system\CodeIgniter.php(832): App\Controllers\Usuarios->insertar()
#5 C:\xampp\htdocs\reportes_cursos\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
#6 C:\xampp\htdocs\reportes_cursos\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\reportes_cursos\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
ERROR - 2024-09-05 16:37:34 --> ErrorException: pg_query(): Query failed: ERROR:  no existe la relación «usuarios»
LINE 2: FROM &quot;usuarios&quot;
             ^ in C:\xampp\htdocs\reportes_cursos\system\Database\Postgre\Connection.php:135
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'C:\\xampp\\htdocs...', 135)
#1 C:\xampp\htdocs\reportes_cursos\system\Database\Postgre\Connection.php(135): pg_query(Resource id #135, 'SELECT 1\nFROM "...')
#2 C:\xampp\htdocs\reportes_cursos\system\Database\BaseConnection.php(670): CodeIgniter\Database\Postgre\Connection->execute('SELECT 1\nFROM "...')
#3 C:\xampp\htdocs\reportes_cursos\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1\nFROM "...')
#4 C:\xampp\htdocs\reportes_cursos\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM "...', Array, false)
#5 C:\xampp\htdocs\reportes_cursos\system\Validation\Rules.php(142): CodeIgniter\Database\BaseBuilder->get()
#6 C:\xampp\htdocs\reportes_cursos\system\Validation\Validation.php(292): CodeIgniter\Validation\Rules->is_unique('tomf', 'usuario', Array, NULL)
#7 C:\xampp\htdocs\reportes_cursos\system\Validation\Validation.php(157): CodeIgniter\Validation\Validation->processRules('usuario', 'usuario', 'tomf', Array, Array)
#8 C:\xampp\htdocs\reportes_cursos\system\Controller.php(160): CodeIgniter\Validation\Validation->run()
#9 C:\xampp\htdocs\reportes_cursos\app\Controllers\Usuarios.php(114): CodeIgniter\Controller->validate(Array)
#10 C:\xampp\htdocs\reportes_cursos\system\CodeIgniter.php(832): App\Controllers\Usuarios->insertar()
#11 C:\xampp\htdocs\reportes_cursos\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
#12 C:\xampp\htdocs\reportes_cursos\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#13 C:\xampp\htdocs\reportes_cursos\public\index.php(37): CodeIgniter\CodeIgniter->run()
#14 {main}
CRITICAL - 2024-09-05 16:37:34 --> Call to a member function getRow() on bool
#0 C:\xampp\htdocs\reportes_cursos\system\Validation\Validation.php(292): CodeIgniter\Validation\Rules->is_unique('tomf', 'usuario', Array, NULL)
#1 C:\xampp\htdocs\reportes_cursos\system\Validation\Validation.php(157): CodeIgniter\Validation\Validation->processRules('usuario', 'usuario', 'tomf', Array, Array)
#2 C:\xampp\htdocs\reportes_cursos\system\Controller.php(160): CodeIgniter\Validation\Validation->run()
#3 C:\xampp\htdocs\reportes_cursos\app\Controllers\Usuarios.php(114): CodeIgniter\Controller->validate(Array)
#4 C:\xampp\htdocs\reportes_cursos\system\CodeIgniter.php(832): App\Controllers\Usuarios->insertar()
#5 C:\xampp\htdocs\reportes_cursos\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
#6 C:\xampp\htdocs\reportes_cursos\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\reportes_cursos\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
ERROR - 2024-09-05 16:38:24 --> ErrorException: pg_query(): Query failed: ERROR:  no existe la relación «usuarios»
LINE 2: FROM &quot;usuarios&quot;
             ^ in C:\xampp\htdocs\reportes_cursos\system\Database\Postgre\Connection.php:135
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'C:\\xampp\\htdocs...', 135)
#1 C:\xampp\htdocs\reportes_cursos\system\Database\Postgre\Connection.php(135): pg_query(Resource id #135, 'SELECT 1\nFROM "...')
#2 C:\xampp\htdocs\reportes_cursos\system\Database\BaseConnection.php(670): CodeIgniter\Database\Postgre\Connection->execute('SELECT 1\nFROM "...')
#3 C:\xampp\htdocs\reportes_cursos\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1\nFROM "...')
#4 C:\xampp\htdocs\reportes_cursos\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM "...', Array, false)
#5 C:\xampp\htdocs\reportes_cursos\system\Validation\Rules.php(142): CodeIgniter\Database\BaseBuilder->get()
#6 C:\xampp\htdocs\reportes_cursos\system\Validation\Validation.php(292): CodeIgniter\Validation\Rules->is_unique('toekdfj2334', 'usuario', Array, NULL)
#7 C:\xampp\htdocs\reportes_cursos\system\Validation\Validation.php(157): CodeIgniter\Validation\Validation->processRules('usuario', 'usuario', 'toekdfj2334', Array, Array)
#8 C:\xampp\htdocs\reportes_cursos\system\Controller.php(160): CodeIgniter\Validation\Validation->run()
#9 C:\xampp\htdocs\reportes_cursos\app\Controllers\Usuarios.php(114): CodeIgniter\Controller->validate(Array)
#10 C:\xampp\htdocs\reportes_cursos\system\CodeIgniter.php(832): App\Controllers\Usuarios->insertar()
#11 C:\xampp\htdocs\reportes_cursos\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
#12 C:\xampp\htdocs\reportes_cursos\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#13 C:\xampp\htdocs\reportes_cursos\public\index.php(37): CodeIgniter\CodeIgniter->run()
#14 {main}
CRITICAL - 2024-09-05 16:38:24 --> Call to a member function getRow() on bool
#0 C:\xampp\htdocs\reportes_cursos\system\Validation\Validation.php(292): CodeIgniter\Validation\Rules->is_unique('toekdfj2334', 'usuario', Array, NULL)
#1 C:\xampp\htdocs\reportes_cursos\system\Validation\Validation.php(157): CodeIgniter\Validation\Validation->processRules('usuario', 'usuario', 'toekdfj2334', Array, Array)
#2 C:\xampp\htdocs\reportes_cursos\system\Controller.php(160): CodeIgniter\Validation\Validation->run()
#3 C:\xampp\htdocs\reportes_cursos\app\Controllers\Usuarios.php(114): CodeIgniter\Controller->validate(Array)
#4 C:\xampp\htdocs\reportes_cursos\system\CodeIgniter.php(832): App\Controllers\Usuarios->insertar()
#5 C:\xampp\htdocs\reportes_cursos\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
#6 C:\xampp\htdocs\reportes_cursos\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\reportes_cursos\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
ERROR - 2024-09-05 16:41:43 --> ErrorException: pg_query(): Query failed: ERROR:  no existe la relación «usuarios»
LINE 2: FROM &quot;usuarios&quot;
             ^ in C:\xampp\htdocs\reportes_cursos\system\Database\Postgre\Connection.php:135
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'C:\\xampp\\htdocs...', 135)
#1 C:\xampp\htdocs\reportes_cursos\system\Database\Postgre\Connection.php(135): pg_query(Resource id #135, 'SELECT 1\nFROM "...')
#2 C:\xampp\htdocs\reportes_cursos\system\Database\BaseConnection.php(670): CodeIgniter\Database\Postgre\Connection->execute('SELECT 1\nFROM "...')
#3 C:\xampp\htdocs\reportes_cursos\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1\nFROM "...')
#4 C:\xampp\htdocs\reportes_cursos\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM "...', Array, false)
#5 C:\xampp\htdocs\reportes_cursos\system\Validation\Rules.php(142): CodeIgniter\Database\BaseBuilder->get()
#6 C:\xampp\htdocs\reportes_cursos\system\Validation\Validation.php(292): CodeIgniter\Validation\Rules->is_unique('toekdfj2334', 'usuario', Array, NULL)
#7 C:\xampp\htdocs\reportes_cursos\system\Validation\Validation.php(157): CodeIgniter\Validation\Validation->processRules('usuario', 'usuario', 'toekdfj2334', Array, Array)
#8 C:\xampp\htdocs\reportes_cursos\system\Controller.php(160): CodeIgniter\Validation\Validation->run()
#9 C:\xampp\htdocs\reportes_cursos\app\Controllers\Usuarios.php(114): CodeIgniter\Controller->validate(Array)
#10 C:\xampp\htdocs\reportes_cursos\system\CodeIgniter.php(832): App\Controllers\Usuarios->insertar()
#11 C:\xampp\htdocs\reportes_cursos\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
#12 C:\xampp\htdocs\reportes_cursos\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#13 C:\xampp\htdocs\reportes_cursos\public\index.php(37): CodeIgniter\CodeIgniter->run()
#14 {main}
CRITICAL - 2024-09-05 16:41:43 --> Call to a member function getRow() on bool
#0 C:\xampp\htdocs\reportes_cursos\system\Validation\Validation.php(292): CodeIgniter\Validation\Rules->is_unique('toekdfj2334', 'usuario', Array, NULL)
#1 C:\xampp\htdocs\reportes_cursos\system\Validation\Validation.php(157): CodeIgniter\Validation\Validation->processRules('usuario', 'usuario', 'toekdfj2334', Array, Array)
#2 C:\xampp\htdocs\reportes_cursos\system\Controller.php(160): CodeIgniter\Validation\Validation->run()
#3 C:\xampp\htdocs\reportes_cursos\app\Controllers\Usuarios.php(114): CodeIgniter\Controller->validate(Array)
#4 C:\xampp\htdocs\reportes_cursos\system\CodeIgniter.php(832): App\Controllers\Usuarios->insertar()
#5 C:\xampp\htdocs\reportes_cursos\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
#6 C:\xampp\htdocs\reportes_cursos\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\reportes_cursos\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
ERROR - 2024-09-05 16:42:49 --> ErrorException: pg_query(): Query failed: ERROR:  no existe la relación «usuarios»
LINE 2: FROM &quot;usuarios&quot;
             ^ in C:\xampp\htdocs\reportes_cursos\system\Database\Postgre\Connection.php:135
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'C:\\xampp\\htdocs...', 135)
#1 C:\xampp\htdocs\reportes_cursos\system\Database\Postgre\Connection.php(135): pg_query(Resource id #135, 'SELECT 1\nFROM "...')
#2 C:\xampp\htdocs\reportes_cursos\system\Database\BaseConnection.php(670): CodeIgniter\Database\Postgre\Connection->execute('SELECT 1\nFROM "...')
#3 C:\xampp\htdocs\reportes_cursos\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1\nFROM "...')
#4 C:\xampp\htdocs\reportes_cursos\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM "...', Array, false)
#5 C:\xampp\htdocs\reportes_cursos\system\Validation\Rules.php(142): CodeIgniter\Database\BaseBuilder->get()
#6 C:\xampp\htdocs\reportes_cursos\system\Validation\Validation.php(292): CodeIgniter\Validation\Rules->is_unique('lkadf93485', 'usuario', Array, NULL)
#7 C:\xampp\htdocs\reportes_cursos\system\Validation\Validation.php(157): CodeIgniter\Validation\Validation->processRules('usuario', 'usuario', 'lkadf93485', Array, Array)
#8 C:\xampp\htdocs\reportes_cursos\system\Controller.php(160): CodeIgniter\Validation\Validation->run()
#9 C:\xampp\htdocs\reportes_cursos\app\Controllers\Usuarios.php(114): CodeIgniter\Controller->validate(Array)
#10 C:\xampp\htdocs\reportes_cursos\system\CodeIgniter.php(832): App\Controllers\Usuarios->insertar()
#11 C:\xampp\htdocs\reportes_cursos\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
#12 C:\xampp\htdocs\reportes_cursos\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#13 C:\xampp\htdocs\reportes_cursos\public\index.php(37): CodeIgniter\CodeIgniter->run()
#14 {main}
CRITICAL - 2024-09-05 16:42:49 --> Call to a member function getRow() on bool
#0 C:\xampp\htdocs\reportes_cursos\system\Validation\Validation.php(292): CodeIgniter\Validation\Rules->is_unique('lkadf93485', 'usuario', Array, NULL)
#1 C:\xampp\htdocs\reportes_cursos\system\Validation\Validation.php(157): CodeIgniter\Validation\Validation->processRules('usuario', 'usuario', 'lkadf93485', Array, Array)
#2 C:\xampp\htdocs\reportes_cursos\system\Controller.php(160): CodeIgniter\Validation\Validation->run()
#3 C:\xampp\htdocs\reportes_cursos\app\Controllers\Usuarios.php(114): CodeIgniter\Controller->validate(Array)
#4 C:\xampp\htdocs\reportes_cursos\system\CodeIgniter.php(832): App\Controllers\Usuarios->insertar()
#5 C:\xampp\htdocs\reportes_cursos\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
#6 C:\xampp\htdocs\reportes_cursos\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\reportes_cursos\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
ERROR - 2024-09-05 16:50:15 --> ErrorException: pg_query(): Query failed: ERROR:  no existe la relación «usuarios»
LINE 2: FROM &quot;usuarios&quot;
             ^ in C:\xampp\htdocs\reportes_cursos\system\Database\Postgre\Connection.php:135
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'C:\\xampp\\htdocs...', 135)
#1 C:\xampp\htdocs\reportes_cursos\system\Database\Postgre\Connection.php(135): pg_query(Resource id #130, 'SELECT 1\nFROM "...')
#2 C:\xampp\htdocs\reportes_cursos\system\Database\BaseConnection.php(670): CodeIgniter\Database\Postgre\Connection->execute('SELECT 1\nFROM "...')
#3 C:\xampp\htdocs\reportes_cursos\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1\nFROM "...')
#4 C:\xampp\htdocs\reportes_cursos\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM "...', Array, false)
#5 C:\xampp\htdocs\reportes_cursos\system\Validation\Rules.php(142): CodeIgniter\Database\BaseBuilder->get()
#6 C:\xampp\htdocs\reportes_cursos\system\Validation\Validation.php(292): CodeIgniter\Validation\Rules->is_unique('ujuan', 'usuario', Array, NULL)
#7 C:\xampp\htdocs\reportes_cursos\system\Validation\Validation.php(157): CodeIgniter\Validation\Validation->processRules('usuario', 'usuario', 'ujuan', Array, Array)
#8 C:\xampp\htdocs\reportes_cursos\system\Controller.php(160): CodeIgniter\Validation\Validation->run()
#9 C:\xampp\htdocs\reportes_cursos\app\Controllers\Prof_cursos.php(65): CodeIgniter\Controller->validate(Array)
#10 C:\xampp\htdocs\reportes_cursos\system\CodeIgniter.php(832): App\Controllers\Prof_cursos->insertar()
#11 C:\xampp\htdocs\reportes_cursos\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Prof_cursos))
#12 C:\xampp\htdocs\reportes_cursos\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#13 C:\xampp\htdocs\reportes_cursos\public\index.php(37): CodeIgniter\CodeIgniter->run()
#14 {main}
CRITICAL - 2024-09-05 16:50:15 --> Call to a member function getRow() on bool
#0 C:\xampp\htdocs\reportes_cursos\system\Validation\Validation.php(292): CodeIgniter\Validation\Rules->is_unique('ujuan', 'usuario', Array, NULL)
#1 C:\xampp\htdocs\reportes_cursos\system\Validation\Validation.php(157): CodeIgniter\Validation\Validation->processRules('usuario', 'usuario', 'ujuan', Array, Array)
#2 C:\xampp\htdocs\reportes_cursos\system\Controller.php(160): CodeIgniter\Validation\Validation->run()
#3 C:\xampp\htdocs\reportes_cursos\app\Controllers\Prof_cursos.php(65): CodeIgniter\Controller->validate(Array)
#4 C:\xampp\htdocs\reportes_cursos\system\CodeIgniter.php(832): App\Controllers\Prof_cursos->insertar()
#5 C:\xampp\htdocs\reportes_cursos\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Prof_cursos))
#6 C:\xampp\htdocs\reportes_cursos\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\reportes_cursos\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
